_posts/2022-08-15-requiring-mfa-on-popular-gems.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Requiring MFA on popular gem maintainers
+layout: post
+author: Jenny Shen
+author_email: jenny.shen@shopify.com
+---
+<p align="center">
+    <img src="/images/gem-with-mfa-flag-dropshadow.png" alt="Doodle of a RubyGem wearing an MFA hat, holding a flag with a checkmark" width="300"/>
+</p>
```

Comment:
Used the same image as the last post. Do we want to change up the doodle this time around to keep it interesting?

Reply:
@bettymakes quickly stirred up another doodle 🚀 , replaced the OG doodle with that one.